docs/madr/decisions/070-resource-identifier.md

```diff
@@ -38,12 +38,245 @@ type TypedResourceIdentifier struct {
+|-------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------|
+| Listener    | `inbound:10.43.205.116:10001`<br>`inbound:[2001:0db8:85a3:0000:0000:8a2e:0370:7334]:10001`                                                                      | ZoneEgress                                     |
+| Cluster     | legacy services - `<mesh>:<kuma.io/service>`<br>new Mesh*Services - `<mesh>_<name>_<namespace>_<zone>_<short-name>_<port>`                                      | Mesh*Service (with sectionName to select port) |
+| FilterChain | for external services - `<kuma.io/service>_<mesh>`                                                                                                              | MeshExternalService                            |
+| Route       | for external services - `outbound:<kuma.io/service>`                                                                                                            | MeshExternalService                            |
+| Secret      | `name       = <category>:<scope>:<identifier>`<br>`category   = "mesh_ca" \| "identity_cert"`<br>`scope      = "secret"`<br>`identifier = "all" \| <mesh_name>` | TODO                                           |
+
+##### MeshPassthrough
+
+|             | Name                                                                                                                                                                                                                                                                                                                                                                                                                               | Correlated Resources |
+|-------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|
+| Cluster     | `meshpassthrough_<protocol>_<match-value>_<port>`<br>when `<port> == 0` Kuma sets port equal to `*`<br>`match-value = CIDR \| IP \| Domain`<br>`CIDR     = i.e. "192.0.2.0/24" or "2001:db8::/32"`<br>`IP       = i.e. "192.0.2.1", or 2001:db8::68", or ::ffff:192.0.2.1"`<br>`Domain   = <dns-name> \| *.<dns-name>`<br>`dns-name = ^([a-zA-Z0-9_]{1}[a-zA-Z0-9_-]{0,62}){1}(\.[a-zA-Z0-9_]{1}[a-zA-Z0-9_-]{0,62})*[\._]?$`<br>  | –                    |
```

Comment:
> passthroughMode: (Optional) Defines behaviour for handling traffic. Allowed values: All, None and Matched. Default: None

What does the identifier should be if `All` and `None` ?

Reply:
yeah, we actually removing them when MeshPassthrough is used https://github.com/kumahq/kuma/blob/6c44719740ef47d1de10564c5db094f946f29989/pkg/plugins/policies/meshpassthrough/plugin/v1alpha1/plugin.go#L98